index.html

```diff
@@ -71,5 +71,141 @@ <h1 id="Greeting-text">Hey there. I’m Shedrack!</h1>
+                  </ul>
+                  <button class="button" type="button">See Project</button>
+              </div>
+          </div>
+          <div class="project-box">
+              <img class="img-project2" alt="project2-image" src="images/placeholder.svg" />
+              <div class="project-details">
+                  <h3>Multi-Post Stories Gain+Glory</h3>
+                  <ul>
+                      <li><a href="#">Ruby on rails</a></li>
+                      <li><a href="#">CSS</a></li>
+                      <li><a href="#">JavaScript</a></li>
+                      <li><a href="#">HTML</a></li>
+                  </ul>
+                  <button class="button" type="button">See Project</button>
+              </div>
+          </div>
+          <div class="project-box">
+              <img class="img-project3" alt="project3-image" src="images/placeholder.svg" />
+              <div class="project-details">
+                  <h3>Multi-Post Stories Gain+Glory</h3>
+                  <ul>
+                      <li><a href="#">Ruby on rails</a></li>
+                      <li><a href="#">CSS</a></li>
+                      <li><a href="#">JavaScript</a></li>
+                      <li><a href="#">HTML</a></li>
+                  </ul>
+                  <button class="button" type="button">See Project</button>
+              </div>
+          </div>
+          <div class="project-box">
+              <img class="img-project4" alt="project4-image" src="images/placeholder.svg" />
+              <div class="project-details">
+                  <h3>Multi-Post Stories Gain+Glory</h3>
+                  <ul>
+                      <li><a href="#">Ruby on rails</a></li>
+                      <li><a href="#">CSS</a></li>
+                      <li><a href="#">JavaScript</a></li>
+                      <li><a href="#">HTML</a></li>
+                  </ul>
+                  <button class="button" type="button">See Project</button>
+              </div>   
+          </div>
+          <div class="project-box">
+              <img class="img-project5" alt="project5-image" src="images/placeholder.svg" />
+              <div class="project-details">
+                  <h3>Multi-Post Stories Gain+Glory</h3>
+                  <ul>
+                      <li><a href="#">Ruby on rails</a></li>
+                      <li><a href="#">CSS</a></li>
+                      <li><a href="#">JavaScript</a></li>
+                      <li><a href="#">HTML</a></li>
+                  </ul>
+                  <button class="button" type="button">See Project</button>
+              </div>
+          </div>
+          
+
+      </section>
+      <section class="about">
+          <p class="name">About me</p>
+          <h3> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+              don’t hestiate to contact me.</h3>
```

Comment:
kindly use a `p` tag considering that this is a paragraph text. This gives a more semantic meaning.